api/routes/game/answer.dart

```diff
@@ -37,6 +37,7 @@ Future<Response> _onPost(RequestContext context) async {
 
   try {
     final (valid, preSolved) = await crosswordRepository.answerWord(
+      user.id,
```

Comment:
Should we specifically catch the new exception and throw a `400` error instead of the now thrown `500`? Since it would be the user doing an bad request and not the server failing?

Reply:
Yes, great idea! I'll make that change.